client/scss/components/_listing.scss

```
@@ -107,6 +107,16 @@ ul.listing {
 
   &.full-width {
     margin-bottom: -3em; // this negates the padding added to the bottom of .content
+
+    thead {
+      position: sticky;
+      top: 140px;
+      @include media-breakpoint-up(sm) {
+        top: 70px;
+      }
+      background-color: theme('colors.surface-page');
+      z-index: 3;
+    }
```

I have not run this code locally yet but in general we should avoid hard-coded values and instead use variables.

We use this `70px` in lots of places, not sure on the best location for this variable but maybe we should add to `client/tailwind.config.js`.

```difff
      addBase({
        /** Support for web components */
        ':root, :host': {
          '--w-font-sans': fontFamily.sans.join(', '),
          '--w-font-mono': fontFamily.mono.join(', '),
          '--w-density-factor': '1',
++          '--w-header-height': '4.375rem', // 70px
```

Then we can update as below.

```suggestion

    thead {
      position: sticky;
      top: calc(var(--w-header-height) * 2);
      @include media-breakpoint-up(sm) {
        top: var(--w-header-height);
      }
      background-color: theme('colors.surface-page');
      z-index: calc(theme('zIndex.header') - 20); // not sure on this but start here and see what works
    }
```